lib/src/legacy/index.ts

```diff
@@ -167,8 +171,24 @@ function convertOptions<sync extends 'sync' | 'async'>(
     verbose: options.verbose,
     charset: options.charset,
     logger: options.logger,
-    legacy: true,
+    legacy: true as const,
   };
+
+  // Use separate return statements, as applying logic directly to the inclusion
+  // of `nodePackageImporter` causes the symbol to use its unique flag.
+  // See https://github.com/microsoft/TypeScript/issues/55901
+  if (options.pkgImporter === 'node') {
+    importers = importers || [];
+    return {
+      ...convertedOptions,
+      importers: [nodePackageImporter, ...importers],
+    };
+  } else {
+    return {
+      ...convertedOptions,
+      importers,
+    };
+  }
```

Comment:
Would it be possible to inject the importer into `importers` before returning the options, so you don't have to create a totally new copy of it here?

Reply:
@jamesnw Do you recall the specific problem caused by the symbol losing its unique flag? I refactored this in https://github.com/sass/embedded-host-node/pull/260/commits/97b3685dbdcbc75f29e0424bd8ff6fd2b4063ff3 and it didn't seem to cause issues that I can see.

Reply:
I think I was trying to construct the importers value outside of the return statement, but didn't try an inline ternary. If you assign a unique symbol to an intermediate const, you lose the unique flag, but it looks like it can be avoided this way.

Reply:
Ah, that makes sense!